packages/toolkit/form-field/form-field-wrapper.selection.css

```
@@ -63,6 +64,79 @@
+  gap: var(--_selection-cluster-gap);
+}
+
+:host(.ngx-signal-form-field-wrapper--selection-cluster)
+  .ngx-signal-form-field-wrapper__prefix,
+:host(.ngx-signal-form-field-wrapper--selection-cluster)
+  .ngx-signal-form-field-wrapper__suffix {
+  display: none;
+}
+
+:host(.ngx-signal-form-field-wrapper--selection-cluster)
+  ::ng-deep
```

Stylelint (17.7.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)